bindings/go/src/ucx/request.go

```diff
@@ -46,18 +47,27 @@ func (p *UcpRequestParams) SetCallback(cb UcpCallback) *UcpRequestParams {
 	return p
 }
 
-func packParams(params *UcpRequestParams, p *C.ucp_request_param_t, cb unsafe.Pointer) uint64 {
+func packArg(cb UcpCallback) unsafe.Pointer {
+	h := cgo.NewHandle(cb)
+	return unsafe.Pointer(&h)
+}
+
+func unpackArg(callback unsafe.Pointer) any {
```

Comment:
let's return UcpCallback

Reply:
I think it's better to return some specific type even if it's interface but maybe we need to come up with a different name than UcpCallback because in some cases we pack objects (bundles) that are not callbacks.

Reply:
let's say callback is such a value that can invoke code and therefore it holds all data necessary for it. I C language it may be pointer to function, in Go it's different, why can't it be some bundle?